tesseract/tesseract_collision/include/tesseract_collision/bullet/bullet_utils.h

```diff
@@ -261,32 +253,47 @@ struct CastHullShape : public btConvexShape
     aabbMax.setMax(max1);
   }
 
+  const char* getName() const override { return "CastHull"; }
+  btVector3 localGetSupportingVertexWithoutMargin(const btVector3& v) const override
+  {
+    return localGetSupportingVertex(v);
+  }
+
+  // LCOV_EXCL_START
+  // notice that the vectors should be unit length
+  void batchedUnitVectorGetSupportingVertexWithoutMargin(const btVector3* /*vectors*/,
+                                                         btVector3* /*supportVerticesOut*/,
+                                                         int /*numVectors*/) const override
+  {
+    throw std::runtime_error("not implemented");
+  }
+
   void getAabbSlow(const btTransform& /*t*/, btVector3& /*aabbMin*/, btVector3& /*aabbMax*/) const override
   {
     throw std::runtime_error("shouldn't happen");
   }
 
-  void setLocalScaling(const btVector3& /*scaling*/) override {}
+  void setLocalScaling(const btVector3& /*scaling*/) override { throw std::runtime_error("shouldn't happen"); }
```

Comment:
This is not a very descriptive error message

Reply:
Yea all of these functions should never get called but must be implemented. This is to catch if anything changes within Bullet.

Reply:
Perhaps we should change the message to something like
"If you are seeing this error message then something in Bullet must have changed. <Insert some idea about what you might do to fix it>"

Reply:
I updated the comment.